.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,7 @@
-# [id:XXX](https://ncarb.tpondemand.com/entity/XXX)
-
 [brief description]
```

Comment:
here I paint the shed `description` -- easy to double-click and type over

Reply:
sure, or not type in at all. But we can see